wordpress-mobile/WordPress-Android
# Fix for choppy scrolling experience in Reader

Fixes #14636 

Reader screen has a CollapsingToolbarLayout and a TabLayout that can feel scrolling up and down at time choppy as reported in the issue.

It is a subtle experience issue.  Seems to be due to using three flags combination on the layout_scrollFlags.  For the curious, this [post](https://medium.com/martinomburajr/android-design-collapsing-toolbar-scrollflags-e1d8a05dcb02) explains in detail.

To test:
Login to WordPress Android app.  Switch to Reader tab from the bottom navigation. Scroll up and down.  Scrolling should be is smoother than before without feeling like stuck when scrolling up or down

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Regular regression tests

3. What automated tests I added (or what prevented me from doing so)
NA

PR submission checklist:
